.github/ISSUE_TEMPLATE/doc_change.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Documentation Request
+about: Tell us how our documentation could improve
+title: "[doc]: A short, simple sentence describing the doc change"
+labels: documentation
+assignees: 'jcalcaben'
+---
+
+<!--
+Thank you for taking the time to report this issue!
+GitHub Issues should only be created for problems/topics related to this project's codebase.
+
+Before submitting this issue, please make sure you are complying with our Code of Conduct:
+https://github.com/magento-research/pwa-studio/blob/develop/.github/CODE_OF_CONDUCT.md
+
+Issues that do not comply with our Code of Conduct or do not contain enough information may be closed at the maintainers' discretion.
+
+Feel free to remove this section before creating this issue.
+-->
+
+**Describe the change**
+A clear and concise description of what change you desire.
+
+**To Reproduce**
+If this is an issue with documentation, provide us a link or steps to get to the
+erroneous page. If this is a new topic request, disregard.
+
+**Screenshots**
+If applicable, add screenshots to help explain the problem.
+
```

Comment:
Add a section for possible solutions. It helps people think through what exactly it is they want.